src/screens/LinkContextScreen/elements/Item.js

```diff
@@ -153,7 +153,7 @@ const styles = StyleSheet.create({
   },
   bottomAnchorSwipeText: {
     fontFamily: fonts.inter[500],
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the color of `bottomAnchorSwipeText` from `COLORS.white` to `COLORS.almostBlack` is also acceptable if it meets the new design specifications. Again, make sure there's enough contrast between this text and its background.

```diff
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->